mycli/main.py

```diff
@@ -111,6 +114,14 @@ def __init__(self, sqlexecute=None, prompt=None,
         # Register custom special commands.
         self.register_special_commands()
 
+        # Load .mylogin.cnf if it exists.
+        mylogin_cnf_path = get_mylogin_cnf_path()
+        if mylogin_cnf_path:
+            mylogin_cnf = get_mylogin_cnf_plaintext(mylogin_cnf_path)
```

Comment:
If there is gibberish in the mylogin.cnf file how will this behave? In other words, you're right we should add some error handling to print a graceful error that says the mylogin.cnf file is corrupted or we're unable to decipher the file.